core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -135,26 +135,35 @@ contract FeePayer is Testable {
         return StoreInterface(finder.getImplementationAddress(storeInterface));
     }
 
-    // The following methods are used by derived classes to interact with collateral that is adjusted by fees.
-    function _getCollateral(FixedPoint.Unsigned storage rawCollateral)
+    // Gets a user-readable collateral value from a raw value that takes fees into account.
```

Comment:
"Returns the raw value of collateral minus any fees that have been subtracted since it was originally deposited into the contract. If any fees have been taken, then the collateral that the user originally deposited will be less than the raw value. "

Reply:
Added a slightly modified version of your suggested comment.